lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -100,6 +100,30 @@ def scrub(transcript)
 
       private
 
+      def add_three_ds(post, payment_method, options)
+        return unless three_d_secure = options[:three_d_secure]
+
+        post[:consumerAuthenticationInformation] ||= {}
+        if payment_method.brand == 'master'
+          post[:consumerAuthenticationInformation][:ucafAuthenticationData] = three_d_secure[:cavv]
+          post[:consumerAuthenticationInformation][:ucafCollectionIndicator] = '2'
```

Comment:
Why are we defaulting this field to `2` ?

Reply:
Great questions! This was taken from our legacy Cybersource integration as the default. 

In terms of the other values under `consumerAuthenticationInformation` Cybersource's docs don't indicate which are required/optional for our purposes. I tried to take the approach of mapping as many of our standard 3DS fields as possible and then cross-referencing with our legacy Cybersource integration to see what data we sent there.

Reply:
Cool, with the passing remote tests I feel pretty good about it. I'll go ahead and 👍